### PR TITLE
Add automatic shell detection if not provided in image/powderaverage

### PR DESCRIPTION
**Pull request by gagnonanthony**
Add automatic shell detection in image/powderaverage. (#41)

### CODE REVIEW

File: modules/nf-neuro/image/powderaverage/main.nf [hunk 1]
```diff
@@ -21,7 +21,7 @@ process IMAGE_POWDERAVERAGE {
     def prefix = task.ext.prefix ?: "${meta.id}"
 
     def b0_thr = task.ext.b0_thr ? "--b0_thr ${task.ext.b0_thr}" : ''
-    def shells = task.ext.shells ? "--shells ${task.ext.shells}" : ''
+    def shells = task.ext.shells ? "--shells ${task.ext.shells}" : "--shells \$(cut -d ' ' --output-delimiter=\$'\\n' -f 1- $bval | awk -F' ' '{v=int(\$1)}{print v}' | uniq)"
```
Comment (AlexVCaron) [inline modules/nf-neuro/image/powderaverage/main.nf:24]: The scilpy script is doing this job itself, when no shell argument is supplied. Why do we need this ?

Comment (gagnonanthony) [reply modules/nf-neuro/image/powderaverage/main.nf:24]: I can't remember exactly why (I should have noted it somewhere), but this fix worked. I'll try to reproduce the error and see what happens. Let's wait for merging in the meantime!

Comment (gagnonanthony) [reply modules/nf-neuro/image/powderaverage/main.nf]: Well, I've tried to reproduce the error again, but it seems to run fine now. I reverted the change @AlexVCaron !

**Comment by AlexVCaron** [comment]
LGTM !

**Pull request merged**